### DISCUSSION TITLE
Copying prod data back to test database

### DISCUSSION THREAD

**Discussion by Ed-M72** [teslamate-org/teslamate#1507] [Q&A]
Am trying to populate some data into the test database, using the production database as a source.  @adriankumpf  - Are there any instructions for this?

I looked at the db backup/restore but the restore won't work as there is no docker-compose.yml file to run the following command against:

docker-compose exec -T database psql -U teslamate

Ed

**Comment by DrMichael** [comment]
Well, if you have postgresql natively running, you could use `psql -U deslamate -d teslamate`

Or what do you mean with "no docker-compose.yml"?

**Reply by Ed-M72** [reply]
Yes, of course you are correct. Brain fart after looking at the screen too long.
